README.md

```diff
@@ -49,3 +53,36 @@ async function handleRequest(request) {
   } else return fetch(request)
 }
 ```
+
+### `serveSinglePageApp`
+
+`serveSinglePageApp` is a custom handler for mapping all "html" requests to the root index.html. For use with Single Page Applications that include in-app routing, such as React Router, VueJS, etc.
+
+#### Example
+
+This example checks the incoming request to see if it evaluates to an html asset, and if so returns the root index.html, otherwise returns the requested asset (e.g. image, css file, js, etc).
+
+```js
+import { getAssetFromKV, serveSinglePageApp } from '@cloudflare/kv-asset-handler'
+
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event.request))
+})
+
+async function handleRequest(request) {
+  if (request.url.includes('/docs')) {
```

Comment:
Do we want to include the prefix example in this example? I think it might further confuse things

Reply:
TBH i just copy-pasted, but i'm open to removing this.